src/Platform/Microsoft.Testing.Platform/Hosts/TestHostBuilder.cs

```diff
@@ -494,6 +493,11 @@ private async Task SendCommandLineOptionsToDotnetTestPipeAsync(NamedPipeClient n
         return namedPipeClient;
     }
 
+    private static bool HasDotnetTestServerOption(CommandLineHandler commandLineHandler) =>
+        commandLineHandler.TryGetOptionArgumentList(PlatformCommandLineProvider.ServerOptionKey, out string[]? serverArgs) &&
+        serverArgs.Length == 1 &&
+        serverArgs[0].Equals(ServerOptionValue, StringComparison.Ordinal);
```

Comment:
Can we rename `ServerOptionValue` in `DotnetTestCliProtocol` or something more clear?